packages/Breadcrumbs/src/Breadcrumbs.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Link from "./components/Link";
+import ExpandButton from "./components/ExpandButton";
+import IsDarkContext from "./context";
+import { MAXIMUM_NUM_OF_LEVEL } from "./constants";
+
+import * as sc from "./Breadcrumbs.styles";
+
+function Breadcrumbs(props) {
+  const { children, isDark, isAutoCollapsed, ...moreProps } = props;
+  const I18n = useI18n();
+  const shouldShowExpandButton = isAutoCollapsed && React.Children.count(children) > MAXIMUM_NUM_OF_LEVEL;
+  const [isCollapsed, setIsCollapsed] = React.useState(shouldShowExpandButton);
+
+  function handleExpand() {
+    setIsCollapsed(false);
+  }
+
+  return (
+    <IsDarkContext.Provider value={isDark}>
+      <sc.Nav role="navigation" aria-label={I18n.t("breadcrumbs.aria_label")} isDark={isDark} {...moreProps}>
```

Comment:
any reason not to use `<nav>`?

Reply:
not really, just want to avoid css conflicts on nav

Reply:
🤷‍♂️ especially in new components, which will be primarily used in new serverless apps, I wouldn't make compromises for fear of global css - if that's what you mean

Reply:
yes! sounds good, I'll change it to a `nav`!

Reply:
Addressed! resolve the conversation